wger/exercises/templates/exercise/view.html

```diff
@@ -92,7 +92,7 @@ <h5 class="mt-4">{% translate "Description" %}:</h5>
 
 {# Images #}
 {% with images=exercise.images.accepted %}
-{% if images or perms.exercises.change_exerciseimage  %}
+{% if images or perms.exercises.change_exerciseimage or user.userprofile.is_trustworthy %}
```

Comment:
We should not allow users to edit or delete images 😅😅, just to submit them!

Reply:
We had to add this check because when we were testing if they were just a normal user and there weren't already images, it never entered this conditional and they weren't able to see the "Add new image" link. When we tested it, they weren't able to edit or delete images because of the nested check on line 107. Does that seem correct?

Reply:
ahh, I see what you mean. I think that since all links are protected with corresponding permission checks, we could simplify this to {% if images %}